prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,72 @@
+"""Taxi Simulator"""
+
+from prac_09.taxi import Taxi
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+
+def main():
```

Comment:
This main function is quite long I would recommend refactoring some parts into functions